src/main/xerus/monstercat/api/Player.kt

```diff
@@ -82,6 +82,12 @@ object Player : FadingHBox(true, targetHeight = 25) {
 		checkFx {
 			showText(text)
 			addButton { resetNotification() }.id("back")
+			if (!Playlist.playlist.isEmpty()) {
+				addButton {
+					val s = if (Playlist.random) Playlist.nextRandom() else Playlist.next()
```

Comment:
the check if it's random or not should be within `Playlist.next()` imo. Maybe make a method `Playlist.playNextItem()` so you can also call it from elsewhere.

Reply:
Fixed